{"name": "Valid value for 'roomSize', 'coords', 'patches' and 'instructions' parameter in the request body -- @1.1 ", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [0, 1], 'patches': 3}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 65, in verify_response_body\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "steps": [{"name": "Given I Set POST request api endpoint", "status": "passed", "start": 1628093067644, "stop": 1628093067645}, {"name": "When I Set HEADER param request content type as \"application/json.\"", "status": "passed", "start": 1628093067645, "stop": 1628093067645}, {"name": "When Set request Body { \"roomSize\" : [2, 2], \"coords\" : [0, 0], \"patches\" : [[0, 0]], \"instructions\" : \"NNESEESWNWW\" }", "status": "passed", "start": 1628093067646, "stop": 1628093067646}, {"name": "And Send a POST HTTP request", "status": "passed", "start": 1628093067646, "stop": 1628093067656}, {"name": "Then I receive valid HTTP response code 200", "status": "passed", "start": 1628093067656, "stop": 1628093067657}, {"name": "And Response BODY with the right data [0, 1] and 1", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [0, 1], 'patches': 3}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 65, in verify_response_body\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "start": 1628093067657, "stop": 1628093067659}], "parameters": [{"name": "request_body", "value": "{ \"roomSize\" : [2, 2], \"coords\" : [0, 0], \"patches\" : [[0, 0]], \"instructions\" : \"NNESEESWNWW\" }"}, {"name": "coords", "value": "[0, 1]"}, {"name": "patches", "value": "1"}], "start": 1628093067642, "stop": 1628093067660, "uuid": "090c9716-b87e-4c28-8144-eeeb874efe6b", "historyId": "04b60b96723c42c45716e966c9e56e36", "fullName": "Test Imaginary Robot for Valid and Invalid Input values for Instructions params: Valid value for 'roomSize', 'coords', 'patches' and 'instructions' parameter in the request body", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Imaginary Robot for Valid and Invalid Input values for Instructions params"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}